### PR TITLE
Keep UI state.

### PR DESCRIPTION
**Pull request by OniOniOn-**
Keeps UI state...
Default sizes are changed for better usability. (picture below)
image

### CODE REVIEW

File: WzComparerR2/MainForm.cs [hunk 1]
```diff
@@ -108,6 +109,78 @@ private void initFields()
...
+                this.columnHeader7.Width = UIStateConfig.Default.ColumnWidth7;
+                this.columnHeader8.Width = UIStateConfig.Default.ColumnWidth8;
+                this.columnHeader9.Width = UIStateConfig.Default.ColumnWidth9;
+                this.dotNetBarManager1.LayoutDefinition = UIStateConfig.Default.BarLayout;
+            }
+            catch(Exception ex)
+            {
+                MessageBox.Show("Invalid ui data!");
+                this.WindowState = FormWindowState.Normal;
+                this.Size = new Size(1200, 800); // = new Size(766, 520);
```
Comment (Kagamia) [inline WzComparerR2/MainForm.cs:169]: what will happen if screen size is small than that size?

File: WzComparerR2/MainForm.cs [hunk 2]
```diff
@@ -108,6 +109,78 @@ private void initFields()
...
+                this.columnHeader5.Width.Absolute = UIStateConfig.Default.ColumnWidth5;
+                this.columnHeader6.Width = UIStateConfig.Default.ColumnWidth6;
+                this.columnHeader7.Width = UIStateConfig.Default.ColumnWidth7;
+                this.columnHeader8.Width = UIStateConfig.Default.ColumnWidth8;
+                this.columnHeader9.Width = UIStateConfig.Default.ColumnWidth9;
+                this.dotNetBarManager1.LayoutDefinition = UIStateConfig.Default.BarLayout;
+            }
+            catch(Exception ex)
+            {
+                MessageBox.Show("Invalid ui data!");
```
Comment (KENNYSOFT) [inline WzComparerR2/MainForm.cs:167]: I think anyone who newly executes this program (without any setting file) will see this message. It's not meaningful in the first launch.

File: WzComparerR2/MainForm.cs (hunk 1 above)
Comment (OniOniOn-) [reply WzComparerR2/MainForm.cs:169]: WcR2 will be bigger than the screen size.

File: WzComparerR2/MainForm.cs (hunk 2 above)
Comment (OniOniOn-) [reply WzComparerR2/MainForm.cs]: Forgot to remove it after testing... Fixed

**Pull request closed without merging**